lib/logger.js

```diff
@@ -39,10 +39,21 @@ function reqIdGenFactory () {
 
 const serializers = {
   req: function asReqValue (req) {
+    var version = req.headers['accept-version']
+    if (version === undefined) {
+      return {
+        id: req.id,
+        method: req.method,
+        url: req.url,
+        remoteAddress: req.connection.remoteAddress,
+        remotePort: req.connection.remotePort
+      }
+    }
```

Comment:
can you remove a bit of duplication from this?

Reply:
Is it ok to log `version: undefined`?